.github/workflows/import.yml

```diff
@@ -31,8 +37,6 @@ jobs:
       - name: Check for changes
         id: git_diff
         run: |
-          git config --global user.name 'github-actions[bot]'
-          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
```

Comment:
I think this had no effect anyway. The create-pull-request action sets the author and committer on its own, and without these the PR creator is shown as `hero-importer (bot)`, which is what I wanted.